### PR TITLE
Added guidance on enrolling to Intune for Jumpbox VM in case of corporate policy requirement

### PR DESCRIPTION
**Pull request by mariekekortsmit**
Added guidance on a private jumpbox VM in case of conditional access policies on trusted devices/Intune to DataManagementAnalytics-ConnectToEnvironmentsPrivately.md

## Summary of the Pull Request
Added guidance on a private jumpbox VM in case of conditional access policies on trusted devices/Intune. Recommended approach is to deploy the VM in your own subscription because of security concerns if you'd put it in a shared subscription. Putting it in your own subscription leads to some additional steps of vnet peering, route table creation, etc. All of which are described in this PR.

## PR Checklist
* [x] Closes Issue #242 
* [x] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/Terminal) and sign the CLA

### CODE REVIEW

File: docs/guidance/DataManagementAnalytics-ConnectToEnvironmentsPrivately.md [hunk 1]
```diff
@@ -68,6 +68,28 @@ If you want to connect to the VM using Azure Bastion, execute the following step
...
 
+### Conditional Access Policies: Enroll your private VM to Intune
+
+If you are deploying your solution into a corporate tenant, you might bump into conditional access policies that enforce you to enroll your VM to Intune for it to be considered a trusted device and access your Azure resources from there. Since working in a shared Azure subscription will grant all subscription owners access to the VM, it is recommended to enroll only VMs in your private subscription to Intune. Please follow the steps below to setup an Intune enrolled Jumpbox VM in your own Azure subscription to connect to the shared Data Management Zone and Data Landing Zone Azure resources. If you are not working in a shared subscription you can safely skip the first steps and go directly to step 6. to enroll the Jumpbox VM created above to Intune.
+
+1. Create a Vnet in your own Azure subscription, make sure that the address space does not overlap with the Vnets for the Data Management Zone `{dmz-prefix}-{environment}-vnet` or Data Landing Zone `{dlz-prefix}-{environment}-vnet` in the shared Azure subscription;
```
Comment (marvinbuss) [inline docs/guidance/DataManagementAnalytics-ConnectToEnvironmentsPrivately.md:75]:
```suggestion
1. Create a Vnet in your own Azure subscription, make sure that the address space does not overlap with the Vnets for the Data Management Zone `{dmz-prefix}-{environment}-vnet` or Data Landing Zone(s) `{dlz-prefix}-{environment}-vnet` in the shared Azure subscription(s);
```

File: docs/guidance/DataManagementAnalytics-ConnectToEnvironmentsPrivately.md [hunk 2]
```diff
@@ -68,6 +68,28 @@ If you want to connect to the VM using Azure Bastion, execute the following step
...
+2. Once created, go the settings pane of the new Vnet and adjust the DNS server to a Custom one with IP address 10.0.0.4 (the IP address that is associated with the Firewall in the Data Management Zone in your shared Azure subscription);
```
Comment (marvinbuss) [inline docs/guidance/DataManagementAnalytics-ConnectToEnvironmentsPrivately.md:76]:
```suggestion
2. Once created, go the settings pane of the new Vnet and adjust the DNS server to the a Custom one (in the default setup the Firewall is used as DNS Forwarder and the default IP address of the Firewall in the Data Management Zone is 10.0.0.4). Check the Vnet of the Data Landing Zone and apply the same settings to your new Vnet;
```

File: docs/guidance/DataManagementAnalytics-ConnectToEnvironmentsPrivately.md [hunk 3]
```diff
@@ -68,6 +68,28 @@ If you want to connect to the VM using Azure Bastion, execute the following step
...
+3. Peer the Vnet to both `{dmz-prefix}-{environment}-vnet` and `{dlz-prefix}-{environment}-vnet` in your shared Azure subscription;
```
Comment (marvinbuss) [inline docs/guidance/DataManagementAnalytics-ConnectToEnvironmentsPrivately.md:77]:
```suggestion
3. Peer the Vnet to both `{dmz-prefix}-{environment}-vnet` and `{dlz-prefix}-{environment}-vnet` in your shared Azure subscription(s);
```

File: docs/guidance/DataManagementAnalytics-ConnectToEnvironmentsPrivately.md [hunk 4]
```diff
@@ -68,6 +68,28 @@ If you want to connect to the VM using Azure Bastion, execute the following step
...
+4. Create a Windows 11 Virtual Machine (Windows 11 Pro - Gen2 image - other images might also work) in  your own Azure subscription and:
+    1. Use your own private admin username and password;
+    2. Close inbound ports;
+    3. Make sure to place the VM inside the newly created Vnet;
+5. Create a new Route Table in your own Azure subscription, with the following setup:
+    1. In Settings > Routes add a default route to firewall with: Address prefix: 0.0.0.0/0, Next hop type: Virtual Appliance, Next hop address: 10.0.0.4 (the IP address that is associated with the Firewall in the Data Management Zone in your shared Azure subscription).
```
Comment (marvinbuss) [inline docs/guidance/DataManagementAnalytics-ConnectToEnvironmentsPrivately.md:83]:
```suggestion
    1. In Settings > Routes add a route with name `default-route-to-firewall` with: Address prefix: 0.0.0.0/0, Next hop type: Virtual Appliance, Next hop address: internal IP of your Firewall (10.0.0.4 is the default internal IP of the Azure Firewall in the Data Management Zone. Please check the Route Table in your Data Landing Zone and apply the same settings).
```

**Review by marvinbuss — CHANGES_REQUESTED** [review]
Great work @mariekekortsmit!
Please check my suggestions below. 👍

**Review by mariekekortsmit — COMMENTED** [review]
Thanks for the feedback! Adjusted it.

**Review by marvinbuss — APPROVED** [review]
LGTM!

**Pull request merged**